### PR TITLE
chore(map): Map cleanup

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

Closes: AVO-611

## Description

Updates the bidding zone borders between the SE subzones to be more accurate, adds the channel islands and removes the NKR zone.

### Double check
- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: config/zones/GG.yaml
Comment (VIKTORVAV99) [inline config/zones/GG.yaml:1]: The translations for this zone already exists.

File: config/zones/JE.yaml
Comment (VIKTORVAV99) [inline config/zones/JE.yaml:1]: Same for this one.

**Comment by VIKTORVAV99** [comment]
> Looks good! Do we want any additional messaging for the channel islands?
> 
> 
> 
> Screenshot 2024-11-06 at 12 05 07
> 
> 

Not just yet, the server error is due to the fact that the backend don't have access to the config files yet.

Once it does it should simply say we don't have any data for it. I'll see if I can make this more graceful in the backend at some point.

**Comment by VIKTORVAV99** [comment]
@tonypls or @madsnedergaard do you have any objections? Otherwise I'll merge this later today.

**Comment by VIKTORVAV99** [comment]
Holding off on merging this until I can do some manual testing with it against the API, shouldn't be any problems but let's just make sure.

**Review by madsnedergaard — APPROVED** [review]
Looks good to me on API side! 🙌 

It changes a 400 to 404 for NRK and adds more zones to /zones, but that's not really a problem - and in the future the /zones endpoint should probably only return zones we actually have data for!

**Comment by VIKTORVAV99** [comment]
Merging this on Monday just to be extra careful.

**Pull request merged**